src/comb/template.clj

```diff
@@ -57,6 +57,6 @@
   ([source]
      (eval source {}))
   ([source bindings]
-     (let [keys (map (comp symbol name) (keys bindings))
+     (let [keys (apply vector (map (comp keyword name) (keys bindings)))
```

Comment:
While it might be an idea to use `mapv` instead of `map` here, this syntax still work in Clojure 1.8.0:

```clojure
nREPL server started on port 58835 on host 127.0.0.1 - nrepl://127.0.0.1:58835
REPL-y 0.3.7, nREPL 0.2.12
Clojure 1.8.0
Java HotSpot(TM) 64-Bit Server VM 1.8.0_25-b17

user=> (let [{:keys (x y)} {:x 1 :y 2}] [x y])
[1 2]
```